pyproject.toml

```diff
@@ -41,7 +41,15 @@ dependencies = [
     "nevergrad",
     "scipy",
     "pyqtorch==1.0.3",
-    "matplotlib"
+    "matplotlib",
+    "jax",
+    "flax",
+    "optax",
+    "jaxopt",
+    "einops",
+    "numpy",
+    "sympy2jax",
+    "horqrux @ git+https://github.com/pasqal-io/horqrux"
```

Comment:
should these be optional?

Reply:
that was being discussed before, if qadence was either full torch or jax, we could do that. but now its kinda both

Reply:
its split now